### DISCUSSION TITLE
disable logs sending to members

### DISCUSSION THREAD

**Discussion by xrtw** [discord-tickets/bot#115] [Support (Q&A)] [answered]
hello, is it possible to disable logs download from members? when  somebody close ticket they can download logs after it

**Comment by eartharoid** [comment, marked as answer]
This can be disabled in `user/config.js`: https://github.com/eartharoid/DiscordTickets/wiki/Configuration#transcriptstextenabled.

If you have further questions, you can join the [Discord support server](https://go.eartharoid.me/discord).
